### ISSUE TITLE
ui-chart: additional lines rendered in gray when adding more than 9 series

### ISSUE THREAD

**Issue by aikitori** [FlowFuse/node-red-dashboard#630]
Hi!

When adding more than 9 series to the chart, the additional lines are displayed in gray.

Is it possible to randomize/brighten/darken the colors, depending on the given color palett?

Anyways, thanks for the new Dashboard. I realy like it, starting to learn some vue for it :)

![grafik](https://github.com/FlowFuse/node-red-dashboard/assets/88290076/f0f36387-c58a-4232-b7fc-53ba430670a0)

**Comment by joepavitt** [comment]
Original Dashboard loops back to the original colours iirc, given we allow for control over the line colours in the Editor, I'm not sure how best to resolve this otherwise. I can introduce randomisation, but then, the user never has control over them? Maybe changes to the colour series editor are required to "add" more beyond the fixed 10 defined?

**Comment by cdunkel1218** [comment]
Is there any known work-arounds?

**Comment by joepavitt** [comment]
I could have sworn @gayanSandamal put a fix in for this? But can't find any evidence. 

@gayanSandamal can you put this onto your todo list please?

**Comment by gayanSandamal** [comment]
> I could have sworn @gayanSandamal put a fix in for this?
> 
> Update: yes, he did - #1220
> 
> Which version are you running @cdunkel1218? You may need to update

Yes, this fix https://github.com/FlowFuse/node-red-dashboard/pull/1220 should fix the above reported issue. 
@cdunkel1218 please let us know if it's still there in the latest version

**Comment by n-lark** [comment]
This was fixed by https://github.com/FlowFuse/node-red-dashboard/pull/1220 - closing out.
